src/graphdb.js

```diff
@@ -52,20 +52,67 @@ function toPascalCase (str) {
     return r.trim();
 }
 
+// Changes every instance of invalid characters in the given label with the following delimiters
+function replaceCleanseLabel(label) {
+    let delimiterColon = "_colon_";
+    let delimiterPeriod = "_period_";
+    let delimiterHyphen = "_hyphen_";
```

Comment:
There are more special characters that need to be replaced: see Punctuators in http://spec.graphql.org/October2021/

> !	$	&	(	)	...	:	=	@	[	]	{	|	}